script.js

```diff
@@ -0,0 +1,198 @@
+import keyCode from './modules/keys.js';
+
+import {
+  domArray,
+  setUpperCaseClass,
+  setLowerCaseClass,
+  insertElement,
+  createElement,
+  setHideAll,
+  setShowAll,
+} from './modules/helpers.js';
+
+import {
+  renderHTML,
+  renderFooter,
+} from './modules/render.js';
+
+import Print from './modules/Print.js';
+import NormalKey from './modules/NormalKey.js';
+
+
+const state = {
+  currentLanguage: 'en',
+  changeLanguage() {
+    if (this.currentLanguage === 'ru') {
```

Comment:
Please use ternary operation here.
Example: this.currentLanguage = this.currentLanguage === 'ru' ? 'en' : 'ru'